rust/builder/mod.rs

```diff
@@ -87,42 +87,50 @@ pub fn var(var: impl Into<UnboundVariable>) -> UnboundVariable {
     var.into()
 }
```

Comment:
Is this ever valid now?

Reply:
We use it in `Sorting`, that can get any `UnboundVariable`.

Reply:
Removed